### DISCUSSION TITLE
Google opinion rewards-可以免費獲得Google play點數的手機App

### DISCUSSION THREAD

**Discussion by a45320614** [1101-GS4719-C/Python-Community-GS4719-C#764] [General]
今天要來分享的是一款可以免費獲得Google play點數的App
Google opinion rewards
![圖片](https://user-images.githubusercontent.com/92359154/195262542-cce5c523-7309-4a5a-8c14-bdc03cd53352.png)
那麼，要如何獲得免費點數?

這個App會不定時發送問卷給使用者，問卷通常有:你有沒有造訪過以下地點，有消費嗎，消費的方式為何等等簡單的問題，都是選擇題，不用1分鐘就可以填答完畢!
而在填答完後就會有一筆點數進入到你的Google帳戶作為你的獎勵囉~
根據我自身的經驗，點數的範圍大概是4到30點，一個月通常就能累積1、200點了~
我通常都拿去買Line貼圖，根本就不用花錢買嘿嘿

這邊再偷偷告訴大家一個冷知識
問卷的發放會以使用者的定位來發送喔，也就是說，你只要到處趴趴走，就能收到一~大堆的問卷，超讚
大家快去下載吧

[Google play頁面連結](https://play.google.com/store/apps/details?id=com.google.android.apps.paidtasks&hl=zh_TW&gl=US)

**Comment by Yunhans** [comment]
太讚了吧但我是用蘋果手機QQ

**Comment by danwan1027** [comment]
太讚了吧，希望蘋果也能推出類似的活動

**Comment by Chiway1229** [comment]
第一次後悔當個果粉了 我也想要免費點數

**Comment by williamyen0623** [comment]
我也好想要酷酷的點數，但我4狗粉。＿。

**Comment by ghost** [comment]
太讚了，感謝分享，是個省錢的好app

**Comment by Pietra1226** [comment]
蘋果什麼時候會有這種好康優惠QQ

**Comment by evanchou1204** [comment]
果粉只能看著app store流口水(._.)

**Comment by Kotori0501** [comment]
太讚了吧 沒想到還有這種方式
這樣想要課金遊戲時，都可以使用免費點數

**Comment by CPCpyt** [comment]
感謝分享，真的非常貼近生活和實用呢！
立馬手刀進入賺取點數。

**Comment by StevenWanglolz** [comment]
appstore可能一輩子都不會有這種優惠 :(
